vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+See `methods(class = "brmsfit")` for more details.
+For example, we may use `summary` to view information about the fitted model, including posterior estimates for the regression coefficients:
+
+```{r}
+summary(fit)
+```
+
+### Sex-stratified model
+
+To fit a model which varies `mu` and `sigma` by sex we alter the `formula` specification to include fixed effects for sex as follows:
+
+```{r}
+fit_sex <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1 + sex, sigma ~ 1 + sex),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+A summary of the model shows that males tend to have longer delays (the posterior mean of `sexM` is `r signif(summary(fit_sex)$fixed[3, "Estimate"], 3)`) and greater delay variation (the posterior mean of `sigma_sexM` is `r signif(summary(fit_sex)$fixed[4, "Estimate"], 3)`):
+
+```{r}
+summary(fit_sex)
+```
+
+### Sex-district stratified model
+
+Finally, we will fit a model which also varies by district.
+To do this, we will use district level random effects, assumed to be drawn from a shared normal distribution, within the model for both the `mu` and `sigma` parameters:
+
+```{r}
+fit_sex_district <- epidist(
+  data = obs_prep,
+  formula = brms::bf(
+    mu ~ 1 + sex + (1 | district),
+    sigma ~ 1 + sex + (1 | district)
+  ),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+Along with looking at the `summary`, we may also use the `brms::ranef` function to look at the estimates of the random effects:
+
+```{r}
+summary(fit_sex_district)
+brms::ranef(fit_sex_district)
+```
+
+## Posterior expectations
+
+To go beyond simple summaries of the fitted model, we recommend using the `tidybayes` package.
+For example, to obtain the posterior expectation of the delay distribution, under no censoring or truncation, we may use the `modelr::data_grid` function in combination with the `tidybayes::add_epred_draws` function.
+The `tidybayes::add_epred_draws` function uses the `posterior_epred_latent_lognormal` function implemented in `epidist` for the `latent_lognormal` custom `brms` family.
+First, for the model with no stratification (Figure \@ref(fig:epred-base)):
+
+```{r}
+epred_draws <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(NA) |>
+  dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA) |>
+  add_epred_draws(fit, dpar = TRUE)
+```
+
+(ref:epred-base) The fitted posterior expectation of the delay distribution for a model with no stratification.
+
+```{r epred-base, fig.cap="(ref:epred-base)"}
+epred_draws |>
+  ggplot(aes(x = .epred)) +
+  tidybayes::stat_slab() +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+Next, for the sex stratified model, we illustrate the higher mean of men as compared with women (Figure \@ref(fig:epred-sex)):
+
+```{r}
+# Probably want functionality for users not to have to do this themselves
```

Comment:
remove comment and make a issue Why does this one have a cut for new data and the others don't?

Reply:
https://github.com/epinowcast/epidist/issues/280